src/meili/routing.cc

```diff
@@ -15,12 +15,11 @@ namespace valhalla {
 
 namespace meili {
 
-LabelSet::LabelSet(const float max_cost, const float bucket_size) {
+LabelSet::LabelSet(const float max_dist, const float bucket_size) {
```

Comment:
Here LabelSet only cares about **cost** regardless it is distance or time, `max_cost` would be a more proper name than `max_dist`.